frontend/app/src/types/transaction.ts

```diff
@@ -71,5 +71,6 @@ export enum TransactionEventProtocol {
   AAVE = 'aave',
   XDAI = 'xdai',
   ZKSYNC = 'zksync',
-  '1INCH' = '1inch'
```

Comment:
Why is the left side of 1inch here also in single quotes?

Reply:
Because variables can't start by a number

Reply:
sure but then its' no longer a variable. At least in a normal language it seems as if you are assigning a constant to a constant. Looks like a syntax error. I assume it's some kind of fucked up hack in javascript?

Reply:
I am still fascinated about how this works. Googling did not help me.

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
or about was it about the quoted enum property thing?

Reply:
It's the quoted variable name!

Reply:
The quoted thing works because the string enum is just an object 

So
```typescript
enum Test {
  '1AE'= '1',
  '2AE'= '2'
}
```
generates the following javascript

```javascript
var Test;
(function (Test) {
    Test["1AE"] = "1";
    Test["2AE"] = "2";
})(Test || (Test = {}));
```

Reply:
oh I see. Thanks a lot @kelsos. So this would only work in typescript, not raw javascript!